disunity-core/src/test/java/info/ata4/util/lz4/SafeUtilsTest.java

```diff
@@ -0,0 +1,127 @@
+package info.ata4.util.lz4;
+
+import com.diffblue.deeptestutils.Reflector;
+import info.ata4.util.lz4.SafeUtils;
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import java.lang.reflect.Array;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+
+public class SafeUtilsTest {
```

Comment:
Would be good to have a non-exceptional case too.

Reply:
Wasn't able to get anything for this.

Reply:
👍  look all good